docs/release_notes/release_notes-0.16.0.rst

```diff
@@ -73,6 +73,7 @@ Key Features and Updates
   * PERF-#4268: Implement partition-parallel __getitem__ for bool Series masks (#4753)
 * Benchmarking enhancements
   * FEAT-#4706: Add Modin ClassLogger to PandasDataframePartitionManager (#4707)
+  * PERF-#4944: Avoid default_to_pandas (#4833)
```

Comment:
We can avoid `default_to_pandas` in many places. Please add the functions for which you do this.